libs/designsystem/src/lib/components/item/item.component.integration.spec.ts

```diff
@@ -2,8 +2,9 @@ import { createHostFactory, SpectatorHost } from '@ngneat/spectator';
 import { IonicModule } from '@ionic/angular';
 
 import { TestHelper } from '../../testing/test-helper';
-import { DesignTokenHelper, WINDOW_TOKEN } from '../../helpers';
+import { WindowRef } from '../../types/window-ref';
 import { ItemComponent } from './item.component';
+import { DesignTokenHelper } from '../../helpers';

```

Comment:
No need to change order:
```suggestion
import { DesignTokenHelper } from '../../helpers';
import { WindowRef } from '../../types/window-ref';
import { ItemComponent } from './item.component';
```